### PR TITLE
Add AdvantageScope to third-party telemetry page

### PR DESCRIPTION
**Pull request by jwbonner**

### CODE REVIEW

**Comment by jwbonner** [comment]
This change closes #2057

**Comment by Starlight220** [comment]
This section isn't very visible. Maybe add a mention in the DataLog section and maybe even the DSLog section?

File: source/docs/software/telemetry/3rd-party-libraries.rst [hunk 1]
```diff
@@ -5,5 +5,6 @@ Third-Party Telemetry Libraries
 
 Several third-party logging utilities and frameworks exist that provide functionality beyond what is currently provided by WPILib:
 
+ * `AdvantageScope <https://github.com/Mechanical-Advantage/AdvantageScope>`__: Data visualization tool for ``NetworkTables``, :ref:`WPILib data logs <docs/software/telemetry/datalog:On-Robot Telemetry Recording Into Data Logs>`, and :ref:`Driver Station logs <docs/software/driverstation/driver-station-log-viewer:Driver Station Log File Viewer>`.
```
Comment (Starlight220) [inline source/docs/software/telemetry/3rd-party-libraries.rst:8]: `:ref:` to NetworkTables as well

File: source/docs/software/telemetry/datalog.rst [hunk 2]
```diff
@@ -219,6 +219,14 @@ After at least one file is loaded, the "Entries" window displays a tree view of
...
 
+Data Log Visualization
+--------------------------
+
+`AdvantageScope <https://github.com/Mechanical-Advantage/AdvantageScope>`__ is a third-party tool allowing users to play back and visualize data stored in WPILib data logs, with support for line graphs, field displays, video synchronization, etc. More details are available in the `AdvantageScope documentation <https://github.com/Mechanical-Advantage/AdvantageScope/blob/main/docs/INDEX.md>`__. Note that WPILib offers no support for third-party projects.
+
+.. image:: images/advantagescope.jpg
```
Comment (sciencewhiz) [inline source/docs/software/telemetry/datalog.rst:227]: As this image is from AdvantageScope which has an MIT license, I believe we'd have to somehow label that this image is MIT licensed.

**Comment by jwbonner** [comment]
Just to avoid any issues with licensing (like whether the full license text needs to be included), I captured a different image that has never been part of AdvantageScope. The new one is also less overwhelming, which I think is a plus.

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**